Makefile

```
@@ -31,6 +31,7 @@ fmt:
 	@gofmt -s -w .
 	@go tool goimports -w .
 	@cd $(ROOT_DIR)/examples && terraform fmt -recursive && cd $(ROOT_DIR)
+	@find $(ROOT_DIR)/stackit/internal/services -type d -name testdata -exec sh -c 'cd "{}" && terraform fmt -recursive' \;
```

```suggestion
	terraform fmt -recursive
```

Can't we just run this on root level? The find seems overkill to me :smile:

---

Also I would like to see a check for this in the CI pipeline maybe: https://github.com/stackitcloud/terraform-provider-stackit/blob/4552b14edda14643b7b1b166f9820fb364f59e03/.github/workflows/ci.yaml